### PR TITLE
Dockerfile for vw binary on alpine image

### PR DESCRIPTION
**Pull request by lalo**
With this image users with Docker installed can easily use vw without having to build or install dependencies.

After building docker image one can use vw as following:

i.e. to display vw command line arguments:
docker run --rm -ti vw-alpine --help

Alpine does not use glibc so vw features depending on glibc won't work on this image for now (open_socket / network.cc). Everything else should work since it's a static link binary.

Resulting image size is 13.6 MB.

repository_dispatch is triggered by POSTing at an endpoint.

### CODE REVIEW

File: vowpal_wabbit/vw-alpine.Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 FROM vowpalwabbit/ubuntu1604-build AS build
-ARG branch_or_tag
+ARG branch_or_tag=8.8.0
```
Comment (jackgerrits) [inline vowpal_wabbit/vw-alpine.Dockerfile]: Is this how you do default values?

**Comment by lalo** [comment]
docker run -v somedir/:/in --rm -ti vw-alpine /in/input.json

File: vowpal_wabbit/vw-alpine.Dockerfile (hunk 1 above)
Comment (lalo) [reply vowpal_wabbit/vw-alpine.Dockerfile]: yea, not a fan of that being the default value but thought it was better than master

Comment (jackgerrits) [reply vowpal_wabbit/vw-alpine.Dockerfile]: Makes sense

**Comment by jackgerrits** [comment]
Can you rename the image to match the scheme here? https://github.com/VowpalWabbit/docker-images/issues/10

**Comment by lalo** [comment]
for a faster experience 

docker run -v /inputdir/:/in --rm -it --entrypoint "/bin/sh" vw-rel-alpine

**Pull request merged**